ghost/core/core/server/services/stats/TopPagesStatsService.js

```
@@ -0,0 +1,209 @@
+                        logging.info('Successfully parsed response body from string');
+                    } catch (e) {
+                        logging.error(`Error parsing response body: ${e.message}`);
+                        responseData = {data: []};
+                    }
+                } else {
+                    responseData = response.body;
+                    logging.info('Using response.body directly as it is not a string');
+                }
+                
+                logging.info(`Response data keys: ${responseData ? JSON.stringify(Object.keys(responseData)) : 'no data'}`);
+                if (responseData && responseData.data && responseData.data.length > 0) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)